src/ucp/api/ucp_def.h

```diff
@@ -385,13 +385,30 @@ typedef void (*ucp_tag_recv_callback_t)(void *request, ucs_status_t status,
  * @brief UCP worker wakeup events mask.
  *
  * The enumeration allows specifying which events are expected on wakeup, though
- * empty events are possible.
+ * empty events are possible for any type of events excepting @ref UCP_WAKEUP_TX
+ * and UCP_WAKEUP_RX which are the most common events.
```

Comment:
Suggested re-wording:
The enumeration allows specifying which events are expected on wakeup. 
Empty events are possible for any type of event except for `@ref`  UCP_WAKEUP_TX and `@ref` UCP_WAKEUP_RX.